A: futebolplayhd.com

edivaniadc opened the pull request:
[Sample URL1](https://futebolplayhd.com/assistir-palmeiras-x-sao-paulo-ao-vivo-gratis-hd-17-08-2021/) 

`||futebolplayhd.com/imagens/logo.png$popup` for any option you click on right after you open the page.

`||doaseeph.com^$popup,third-party` for server _Opção 9_ (it didn't work for me with $third-party only)

Redirects for server _Opção 11_ 
```
||azcdn107.net^$third-party - ask to download a file (tested using US and BR VPN)
||ihogaetw.com^$third-party  
||sdxqusoze.com^$third-party
||qjcdnpv.com^$third-party

```
<img width="1326" alt="fths" src="https://user-images.githubusercontent.com/57706597/132234090-dc665168-fe87-487d-8e04-6193372f749a.png">

jabonsolo commented:
For the popups I think `||doaseeph.com^$popup,third-party` would be enough. 

`||futebolplayhd.com/imagens/logo.png$popup` I found it unnecessary, that could be removed.

The other stuff you mentioned couldn't reproduce, or didn't know how to (tried with VPN), if you could elaborate how to reproduce, please.

edivaniadc commented:
@jabonsolo thanks for the review and sorry for the delay. 
I deleted `||futebolplayhd.com/imagens/logo.png$popup` from easylistportuguese_specific_block_popup.txt on [commit](https://github.com/edivaniadc/easylistportuguese/commit/4431a97)

Regarding the other stuff I found `29hm3pwm68.azcdn107.com` and it was redirecting to a download file, example:
<img width="1381" alt="fhd" src="https://user-images.githubusercontent.com/57706597/132659978-9fc60a9a-9a8a-4a0a-80ed-a14914a41280.png"> so I added `||azcdn107.net^$third-party`. I tried now again with US and BR vpn and could no longer get hit of it. The other below rules I was getting redirects. May I delete all these rules tho?
```
||azcdn107.net^$third-party
||ihogaetw.com^$third-party
||sdxqusoze.com^$third-party
||qjcdnpv.com^$third-party
```

jabonsolo commented:
We can keep those ones. Thanks
